include/aca_util.h

```diff
@@ -105,6 +109,46 @@ static inline const char *aca_get_neighbor_type_string(alcor::schema::NeighborTy
   }
 }
 
+static inline bool aca_validate_mac_address(const char *mac_string)
+{
+  unsigned char mac[6];
+
+  if (mac_string == nullptr) {
+    ACA_LOG_ERROR("Input mac_string is null\n");
+    return false;
+  }
+
+  if (sscanf(mac_string, "%hhx:%hhx:%hhx:%hhx:%hhx:%hhx", &mac[0], &mac[1],
+             &mac[2], &mac[3], &mac[4], &mac[5]) == 6) {
+    return true;
+  }
+
+  if (sscanf(mac_string, "%hhx-%hhx-%hhx-%hhx-%hhx-%hhx", &mac[0], &mac[1],
+             &mac[2], &mac[3], &mac[4], &mac[5]) == 6) {
+    return true;
+  }
+
+  // nothing matched
+  ACA_LOG_ERROR("Invalid mac address: %s\n", mac_string);
+  return false;
+}
+
+static inline bool aca_validate_tunnel_id(const uint tunnel_id)
```

Comment:
Add a new parameter network_type so that we know where to update when we plan to support new network_type other than VxLan.

Reply:
fixed.